less/navigation.less

```diff
@@ -1 +1,49 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+
+.navigation {
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 10rem;
+    position: fixed;
+    top:0;
+    background-color: whitesmoke;
+    width: 100%;
+    opacity: 0.95;
+    
+    
+  
+    h1 {
+      .headerOne();
+    }
+  
+    nav {
+        .navigator();
+  
+      a {
+        font-size: 1.9rem;
+        text-decoration: none;
+        color: black;
+        padding: 1.5rem;
+      
+      
+  
+      a:hover {
+        color: black;
+        font-weight: bolder;
+      }
+    }
+  
+    @media @phone {
+      h1 {
+          .headerOne();
+        
```

Comment:
is this second h1 selection necessary? How would the cascade effect from line 17-19 effect this?

Reply:
It's already applied above....as above - so below.  Not necessary. I do this sometimes just to know where I am on the page.  I should use comments instead. That would make more sense.